githubbot/githubbot/http.go

```diff
@@ -212,7 +229,10 @@ func (h *HTTPSrv) formatMessage(event interface{}, repo string, defaultBranch st
 		if len(pullRequests) == 0 && len(event.Branches) >= 1 {
 			// this is a branch test, not associated with a PR
 			branch = event.Branches[0].GetName()
+			author = getPossibleKBUser(h.kbc, h.db, h.DebugOutput, event.GetCommit().GetAuthor().GetLogin())
 		}
+
+		author = getPossibleKBUser(h.kbc, h.db, h.DebugOutput, pullRequests[0].GetUser().GetLogin())
```

Comment:
If there are no pull requests do it still hit this line?
Maybe put it in the `else` of the above `if` statement?

Reply:
oh whoops, good catch!